### ISSUE TITLE
`duc info` returns zero exitcode (success), even if the database is missing or corrupt

### ISSUE THREAD

**Issue by sumpfralle** [zevv/duc#303]
The `duc info` command does not complain (via its exitcode), if the given `--database` filename does not refer to an existing file (same effect if the database exists and is corrupt).

```shell
$ duc info --database foo.db; echo $?
Error opening: foo.db - Database not found
Database not found
0
```

Above you see, that the command returns the exitcode zero.
Instead, it should return a non-zero exitcode.

**Comment by zevv** [comment]
This has been fixed in commit 2ab7d1c0f530e9b2686b2a9dd7fdd381b8a6f890,
released with duc v1.4.5



--
:wq
^X^Cy^K^X^C^C^C^C

**Comment by sumpfralle** [comment]
Ah - sorry, that I omitted the version I use.
Thank you!
